test/D2L.Security.OAuth2.WebApi.IntegrationTests/Authorization/NoImpersonationAttributeTests.cs

```diff
@@ -29,14 +29,24 @@ await TestUtilities
 		}
 
 		[Test]
-		public async Task ImpersonationToken_Forbidden() {
+		public async Task ImpersonationToken_Unauthorized() {
 			string jwt = await TestUtilities
 				.GetAccessTokenValidForAMinute( userId: 1234, actualUserId: 1235 )
 				.SafeAsync();
 
-			await TestUtilities
-				.RunBasicAuthTest( "/authorization/imp", jwt, HttpStatusCode.Forbidden )
+			var response = await TestUtilities
+				.RunBasicAuthTest<OAuth2ErrorResponse>( "/authorization/imp", jwt, HttpStatusCode.Unauthorized )

```

Comment:
I changed these cause of the RFC suggestion, and I didn't think it super terribly mattered.

```
invalid_token
         The access token provided is expired, revoked, malformed, or
         invalid for other reasons.  The resource SHOULD respond with
         the HTTP 401 (Unauthorized) status code.  The client MAY
         request a new access token and retry the protected resource
         request.
```